app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -91,7 +167,8 @@ $image-overlay-form-width: 350px;
     left: 50% !important;
     top: 50% !important;
     @include transform(translateY(-50%));
-    margin-left: -$image-overlay-form-width/2;
+    margin-left: -$image-overlay-form-width/2 + $picture-overlay-handle-width;
```

Comment:
`$image-overlay-form-width/2` should be written with a single space on each side of the operator: `$image-overlay-form-width / 2`